app/package.json

```diff
@@ -17,7 +17,7 @@
     "@babel/compat-data": "^7.24.4",
     "@bcrs-shared-components/breadcrumb": "2.1.24",
     "@bcrs-shared-components/corp-type-module": "1.0.15",
-    "@bcrs-shared-components/enums": "1.1.7",
+    "@bcrs-shared-components/enums": "1.1.13",
```

Comment:
This is to pull in the latest NR request type codes (as added in bcgov/entity#23382).

These codes are not used in Namerequest UI but it's good to keep things up to date.